cmd/kube-apiserver/app/server.go

```diff
@@ -315,10 +315,11 @@ func CreateKubeAPIServerConfig(s *options.ServerRunOptions, nodeTunneler tunnele
 
 		ServiceIPRange:       serviceIPRange,
 		APIServerServiceIP:   apiServerServiceIP,
-		APIServerServicePort: 443,
+		APIServerServicePort: s.APIServerServicePort,
 
-		ServiceNodePortRange:      s.ServiceNodePortRange,
-		KubernetesServiceNodePort: s.KubernetesServiceNodePort,
+		ServiceNodePortRange:          s.ServiceNodePortRange,
+		KubernetesServiceNodePort:     s.KubernetesServiceNodePort,
+		KubernetesServiceExternalName: s.KubernetesServiceExternalName,
```

Comment:
1. grouping: group the service IP range and node port range together, and the kubernetes/apiserver service stuff together.

2. naming: Consistently use either APIServerService or KubernetesService, it doesn't matter which but let's use a single word for this concept or it will be confusing. Argh, I guess we already had both names. Can we standardize on "Kubernetes" as part of this change? Sorry to make you do extra work, but my rule for the quarter is only changes that ara net reductions in tech debt :)